src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java

```diff
@@ -1,9 +1,11 @@
 package com.minecolonies.coremod.util;
 
+import com.ldtteam.structurize.util.LanguageHandler;
 import com.minecolonies.api.colony.IChunkmanagerCapability;
```

Comment:
can you undo the reformat too?

Reply:
Yeah, but I thought the goal was to get all the code to a specific format?

Reply:
yeah, but it's difficult to see changes in a PR with that